buildspec.yml

```
@@ -0,0 +1,33 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      - git clone https://github.com/crungehottman/exodus-lambda.git
+      - cd exodus-lambda
+      - git checkout codebuild
+      - gpg --import .whitelist.gpg
+      - LAST_TAGGED=$(git rev-list --tags --max-count=1)
+      - git verify-tag $(git describe $LAST_TAGGED)
+      - git checkout $LAST_TAGGED
+      - pip install --require-hashes -r requirements.txt --target ./package
+      - pip install --no-deps --target ./package .
+      - cp ./configuration/exodus-lambda-deploy.yaml ./package
+      - >-
+        jq -n "{
+           \"table\": {
+             \"name\": \"exodus-cdn-$ENV_TYPE\", region: \"us-east-1\"
+           },
+           \"headers\": {
+             \"max-age\": \"600\"
+           }
+         }" > ./package/lambda_config.json
+      - aws cloudformation package
+        --template ./package/exodus-lambda-deploy.yaml
+        --s3-bucket exodus-pipeline-artifacts
+        --output-template-file ./package/exodus-lambda-pkg.yaml
+
+artifacts:
+  files:
+    - ./exodus-lambda/package/lambda_config.json
+    - ./exodus-lambda/package/exodus-lambda-pkg.yaml
```

We shouldn't need lambda_config.json exported. It's already in the package and available to the function.
Also, it would be nice to have `discard-paths` option here so we don't need to use the whole path when referencing files later on.
```suggestion
artifacts:
  files:
    - ./exodus-lambda/package/exodus-lambda-pkg.yaml
  discard-paths: yes
```